### DISCUSSION TITLE
Station setup menu and views are different for different users. Why?

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#661] [Q&A] [answered]
I created a test user in my Wavelog instance (50RS074 is my callsign in the local CB DX club) and noticed that the appearance of the User menu (under Callsign) and the Station Setup menu for this user are different than when I am logged in with my primary callsign (R1BLH).

In particular, station setup and location selection are displayed in a separate menu in the case of the primary callsign, and in this case it is possible to add some locations to favorites.
![изображение](https://github.com/user-attachments/assets/a5b4a696-f9ae-4cf3-9bd4-26ea0bbef6dd)
![изображение](https://github.com/user-attachments/assets/23ebb32e-2113-4086-b827-1aa8a390e88f)
![изображение](https://github.com/user-attachments/assets/cd3f1e07-e36a-429b-ae50-7cf0929ee5d6)

In the case of the other callsign, the station setup menu is located in the general User menu, and there is no option to select favorite locations.
![изображение](https://github.com/user-attachments/assets/eb03a10b-cd8e-4d38-aaf3-4fec678f53dc)
![изображение](https://github.com/user-attachments/assets/32612ff8-2ff0-4dff-a7a9-ea66a332ee60)

What could be the reason for this and how can I fix it?

**Comment by HB9HIL** [comment, marked as answer]
Check the User Setting "Station Locations Quickswitch" in the "Wavelog Preferences" Tab 

(Russian Option: "Быстрое переключение профилей QTH" in "Настройки Wavelog")

**Comment by R1BLH** [comment]
Thank you! I missed this.
